libioc/Types.py

```diff
@@ -27,20 +27,47 @@
 import re
 
 
-class AbsolutePath(str):
+class Path(str):
```

Comment:
I'm not happy with this
i think we should be using https://docs.python.org/3/library/pathlib.html instead of string

Reply:
We can address this suggestion in #617.